specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/compute.json

```diff
@@ -10918,4 +10920,4 @@
       "description": "Client Api Version."
     }
   }
-}
+}
```

Comment:
This will cause prettier check failure.

Reply:
What does it want at the end? A space? A line feed?